packages/math-input/src/components/iconography/left.js

```diff
@@ -1,7 +1,7 @@
 /**
  * An component that renders the LEFT iconograpy in SVG.
  */
-const React = require("react");
+import React from "react";
```

Comment:
These should be updated to `import * as React from "react"` when adopting static typing, but this is fine for now.

Reply:
I've fixed this in the next PR.